test/integration/test_integration.py

```diff
@@ -102,7 +102,9 @@
 def _int8wo_api(mod):
     if TORCH_VERSION_AT_LEAST_2_4:
         quantize_(mod, int8_weight_only(), set_inductor_config=False)
-        if not TORCH_VERSION_AT_LEAST_2_5:
+        if (
+            not TORCH_VERSION_AT_LEAST_2_5
+        ) or torch._inductor.config.freezing:
```

Comment:
when is `freezing` set? how do user know that they need to call `unwrap_tensor_subclass` in that case?

Reply:
Hi Jerry, `freezing` is setting in https://github.com/pytorch/ao/blob/8236a874479a9a9168e584c81dda8707f4c41006/test/integration/test_integration.py#L829 when running the UT. User can also set it as `torch._inductor.config.freezing=True` in script or using the environment variable as `TORCHINDUCTOR_FREEZING=1`

Reply:
> how do user know that they need to call unwrap_tensor_subclass in that case?

That's a good idea. Did you mean we should write document somewhere?

Reply:
I see, thanks for the context! yeah I think it would be good to add this in the README: https://github.com/pytorch/ao/tree/main/torchao/quantization#workaround-with-unwrap_tensor_subclass-for-export-aoti-and-torchcompile-pytorch-24-and-before-only and link the issue as well

Reply:
Thanks and adding the readme with issue link in this PR. please help to take a look again.